From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Sun, 13 Jan 2013 03:51:50 +0100
Subject: small change. compile now with gcc-4.7

---
 libs/qCC_db/ccChunkedArray.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/qCC_db/ccChunkedArray.h b/libs/qCC_db/ccChunkedArray.h
index d9f3419..46f43ce 100644
--- a/libs/qCC_db/ccChunkedArray.h
+++ b/libs/qCC_db/ccChunkedArray.h
@@ -48,7 +48,7 @@ public:
 	virtual ccChunkedArray<N,ScalarType>* clone()
 	{
 		ccChunkedArray<N,ScalarType>* cloneArray = new ccChunkedArray<N,ScalarType>(getName());
-		if (!copy(*cloneArray))
+		if (!this->copy(*cloneArray))
 		{
 			ccLog::Error("[ccChunkedArray::clone] Failed to clone array (not enough memory?)");
 			cloneArray->release();
