From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Mon, 3 Nov 2014 00:36:31 +0100
Subject: LIBLAS_support

---
 contrib/LiblasSupport.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/contrib/LiblasSupport.cmake b/contrib/LiblasSupport.cmake
index aec413a..84b8b8d 100644
--- a/contrib/LiblasSupport.cmake
+++ b/contrib/LiblasSupport.cmake
@@ -24,8 +24,8 @@ if( ${OPTION_USE_LIBLAS} )
 	mark_as_advanced(CLEAR Boost_LIBRARY_DIRS) 
 
 	# liblas
-	set( LIBLAS_INCLUDE_DIR "" CACHE PATH "LibLAS include directory" )
-	set( LIBLAS_RELEASE_LIBRARY_FILE "" CACHE FILEPATH "LibLAS library file (release mode)" )
+	set( LIBLAS_INCLUDE_DIR "/usr/include" CACHE PATH "LibLAS include directory" )
+	set( LIBLAS_RELEASE_LIBRARY_FILE "/usr/lib/liblas.so" CACHE FILEPATH "LibLAS library file (release mode)" )
 	if (WIN32)
 		set( LIBLAS_SHARED_LIBRARY_FILE "" CACHE FILEPATH "LibLAS shared library file (dll release mode)" )
 		set( LIBLAS_DEBUG_LIBRARY_FILE "" CACHE FILEPATH "LibLAS library file (debug mode)" )
