From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Mon, 14 Jan 2013 20:50:53 +0100
Subject: Aux files copy is not handled at cmake level
--- a/qCC/CMakeLists.txt
+++ b/qCC/CMakeLists.txt
@@ -122,7 +122,7 @@ endif()
 install_ext( TARGETS ${PROJECT_NAME} ${CLOUDCOMPARE_DEST_FOLDER} )
 
 # Auxiliary files
-set( auxFiles bin_other/history.txt bin_other/license.txt bin_other/global_shift_list_template.txt )
+# set( auxFiles bin_other/history.txt bin_other/license.txt bin_other/global_shift_list_template.txt )
 
 if( WIN32 )
 	# Export Qt dlls
@@ -135,9 +135,9 @@ if( WIN32 )
 endif()
 
 # Install auxiliary files
-foreach( filename ${auxFiles} )
-	install_ext( FILES ${filename} ${CLOUDCOMPARE_DEST_FOLDER} )
-endforeach()
+# foreach( filename ${auxFiles} )
+#	install_ext( FILES ${filename} ${CLOUDCOMPARE_DEST_FOLDER} )
+# endforeach()
 
 # in order to ensure that everything else is installed first, put the Mac bundling in its own subdirectory
 if( APPLE )
--- a/ccViewer/CMakeLists.txt
+++ b/ccViewer/CMakeLists.txt
@@ -130,9 +130,9 @@ if( WIN32 )
 endif()
 
 # Install auxiliary files
-foreach( filename ${auxFiles} )
-	install_ext( FILES ${filename} ${CCVIEWER_DEST_FOLDER} )
-endforeach()
+# foreach( filename ${auxFiles} )
+#	install_ext( FILES ${filename} ${CCVIEWER_DEST_FOLDER} )
+# endforeach()
 
 # Put together our App bundle on Mac OS X
 if( APPLE )
