From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Mon, 14 Jan 2013 21:15:12 +0100
Subject: Plugins and shaders paths
--- a/libs/qCC_glWindow/ccGLWindow.cpp
+++ b/libs/qCC_glWindow/ccGLWindow.cpp
@@ -3935,7 +3935,7 @@ QString ccGLWindow::getShadersPath()
 	path.remove( "MacOS" );
 	return path + "Shaders";
 #else
-	return QApplication::applicationDirPath() + "/shaders";
+	return "/usr/share/cloudcompare/shaders/CloudCompare";
 #endif
 }
 
--- a/qCC/mainwindow.cpp
+++ b/qCC/mainwindow.cpp
@@ -395,7 +395,7 @@ void MainWindow::loadPlugins()
 	m_pluginsPath = path + "Plugins/ccPlugins";
 #else
 	//plugins are in bin/plugins
-	m_pluginsPath = QCoreApplication::applicationDirPath()+QString("/plugins");
+	m_pluginsPath = "/usr/lib/cloudcompare/plugins/CloudCompare";
 #endif
 
 	ccConsole::Print(QString("Plugins lookup dir.: %1").arg(m_pluginsPath));
--- a/ccViewer/ccviewer.cpp
+++ b/ccViewer/ccviewer.cpp
@@ -190,7 +190,7 @@ void ccViewer::loadPlugins()
 	QString pluginsPath = path + "Plugins/ccViewerPlugins";
 #else
 	//plugins are in bin/plugins
-	QString pluginsPath = QCoreApplication::applicationDirPath()+QString("/plugins");
+	QString pluginsPath = "/usr/lib/cloudcompare/plugins/ccViewer";
 #endif
 
 	ccLog::Print(QString("Plugins lookup dir.: %1").arg(pluginsPath));
--- a/qCC/CMakeLists.txt
+++ b/qCC/CMakeLists.txt
@@ -120,7 +120,7 @@ if( WIN32 )
 endif()
 
 # install program
-install_ext( TARGETS ${PROJECT_NAME} ${CLOUDCOMPARE_DEST_FOLDER} )
+install_ext( TARGETS ${PROJECT_NAME} /usr/bin )
 
 # Auxiliary files
 # set( auxFiles bin_other/history.txt bin_other/license.txt bin_other/global_shift_list_template.txt )
@@ -160,9 +160,9 @@ if( APPLE )
    #install( FILES ${CMAKE_CURRENT_SOURCE_DIR}/shaders/Rendering/rendering.frag DESTINATION ${CLOUDCOMPARE_MAC_BASE_DIR}/Contents/Shaders/Rendering )
    #install( FILES ${CMAKE_CURRENT_SOURCE_DIR}/shaders/Rendering/rendering.vert DESTINATION ${CLOUDCOMPARE_MAC_BASE_DIR}/Contents/Shaders/Rendering )
 else()
-   install_ext( FILES ${CC_FBO_LIB_SOURCE_DIR}/bilateral/bilateral.frag ${CLOUDCOMPARE_DEST_FOLDER} /shaders )
-   install_ext( FILES ${CC_FBO_LIB_SOURCE_DIR}/bilateral/bilateral.vert ${CLOUDCOMPARE_DEST_FOLDER} /shaders )
-   install_ext( FILES ${CMAKE_CURRENT_SOURCE_DIR}/shaders/ColorRamp/color_ramp.frag ${CLOUDCOMPARE_DEST_FOLDER} /shaders/ColorRamp )
+    install( FILES ${CC_FBO_LIB_SOURCE_DIR}/bilateral/bilateral.frag DESTINATION /usr/share/cloudcompare/shaders/CloudCompare )
+    install( FILES ${CC_FBO_LIB_SOURCE_DIR}/bilateral/bilateral.vert DESTINATION /usr/share/cloudcompare/shaders/CloudCompare )
+    install( FILES ${CMAKE_CURRENT_SOURCE_DIR}/shaders/ColorRamp/color_ramp.frag DESTINATION /usr/share/cloudcompare/shaders/CloudCompare/ColorRamp )
    #install_ext( FILES ${CMAKE_CURRENT_SOURCE_DIR}/shaders/Rendering/rendering.frag ${CLOUDCOMPARE_DEST_FOLDER} /shaders/Rendering )
    #install_ext( FILES ${CMAKE_CURRENT_SOURCE_DIR}/shaders/Rendering/rendering.vert ${CLOUDCOMPARE_DEST_FOLDER} /shaders/Rendering )
 endif()
--- a/ccViewer/CMakeLists.txt
+++ b/ccViewer/CMakeLists.txt
@@ -116,7 +116,7 @@ if( WIN32 )
 endif()
 
 # install program
-install_ext( TARGETS ccViewer ${CCVIEWER_DEST_FOLDER} )
+install_ext( TARGETS ccViewer /usr/bin )
 
 # Auxiliary files
 set( auxFiles bin_other/license.txt )
@@ -154,5 +154,5 @@ endif( APPLE )
 if( APPLE )
    install( FILES ${CMAKE_CURRENT_SOURCE_DIR}/../qCC/shaders/ColorRamp/color_ramp.frag DESTINATION ${CCVIEWER_MAC_BASE_DIR}/Contents/Shaders/ColorRamp )
 else()
-   install_ext( FILES ${CMAKE_CURRENT_SOURCE_DIR}/../qCC/shaders/ColorRamp/color_ramp.frag ${CCVIEWER_DEST_FOLDER} /shaders/ColorRamp )
+    install( FILES ${CMAKE_CURRENT_SOURCE_DIR}/../qCC/shaders/ColorRamp/color_ramp.frag DESTINATION /usr/share/cloudcompare/shaders/ccViewer/ColorRamp )
 endif()
--- a/plugins/CMakePluginTpl.cmake
+++ b/plugins/CMakePluginTpl.cmake
@@ -79,7 +79,7 @@ if( APPLE )
 	install( TARGETS ${PROJECT_NAME} LIBRARY DESTINATION ${CLOUDCOMPARE_MAC_BASE_DIR}/Contents/Plugins/ccPlugins COMPONENT Runtime )
 	set( CLOUDCOMPARE_PLUGINS ${CLOUDCOMPARE_PLUGINS} ${CLOUDCOMPARE_MAC_BASE_DIR}/Contents/Plugins/ccPlugins/lib${PROJECT_NAME}${CMAKE_SHARED_LIBRARY_SUFFIX} CACHE INTERNAL "CloudCompare plugin list")
 else()
-	install_shared( ${PROJECT_NAME} ${CLOUDCOMPARE_DEST_FOLDER} 1 /plugins )
+	install_shared( ${PROJECT_NAME} /usr/lib/cloudcompare/plugins 1 /CloudCompare )
 endif()
 
 #GL filters and IO plugins also go the the ccViewer 'plugins' sub-folder
@@ -90,7 +90,7 @@ if( ${OPTION_BUILD_CCVIEWER} )
 			install( TARGETS ${PROJECT_NAME} LIBRARY DESTINATION ${CCVIEWER_MAC_BASE_DIR}/Contents/Plugins/ccViewerPlugins COMPONENT Runtime )
 			set( CCVIEWER_PLUGINS ${CCVIEWER_PLUGINS} ${CCVIEWER_MAC_BASE_DIR}/Contents/Plugins/ccViewerPlugins/lib${PROJECT_NAME}${CMAKE_SHARED_LIBRARY_SUFFIX} CACHE INTERNAL "ccViewer plugin list")
 		else()
-			install_shared( ${PROJECT_NAME} ${CCVIEWER_DEST_FOLDER} 1 /plugins )
+			install_shared( ${PROJECT_NAME} /usr/lib/cloudcompare/plugins 1 /ccViewer )
 		endif()
 	endif()
 
@@ -108,9 +108,9 @@ if( CC_SHADER_FOLDER )
 				install( FILES ${filename} DESTINATION ${CCVIEWER_MAC_BASE_DIR}/Contents/Shaders/${CC_SHADER_FOLDER} )
 			endif()
 		else()
-			install_ext( FILES ${filename} ${CLOUDCOMPARE_DEST_FOLDER} /shaders/${CC_SHADER_FOLDER} )
+            install( FILES ${filename} DESTINATION /usr/share/cloudcompare/shaders/CloudCompare/${CC_SHADER_FOLDER} )
 			if( ${OPTION_BUILD_CCVIEWER} )
-				install_ext( FILES ${filename} ${CCVIEWER_DEST_FOLDER} /shaders/${CC_SHADER_FOLDER} )
+                install( FILES ${filename} DESTINATION /usr/share/cloudcompare/shaders/ccViewer/${CC_SHADER_FOLDER} )
 			endif()
 		endif()
 	endforeach()
