From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Mon, 14 Jan 2013 21:15:12 +0100
Subject: Use debian libGLEW
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -59,7 +59,7 @@ include( CMakeExternalLibs.cmake )
 include( contrib/AllSupport.cmake )
 
 # Libs requested by both qCC & ccViewer
-add_subdirectory( libs/Glew )
+# add_subdirectory( libs/Glew )
 add_subdirectory( libs/CCFbo )
 add_subdirectory( libs/qCC_db ) # must always be included after CCFbo (dependency)
 add_subdirectory( libs/qCC_io ) # must always be included after qCC_db (dependency)
--- a/ccViewer/CMakeLists.txt
+++ b/ccViewer/CMakeLists.txt
@@ -9,7 +9,7 @@ if(POLICY CMP0020)
   cmake_policy(SET CMP0020 OLD)
 endif()
 
-include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
+# include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
 include_directories( ${CC_FBO_LIB_SOURCE_DIR}/include )
 include_directories( ${CC_CORE_LIB_SOURCE_DIR}/include )
 include_directories( ${QCC_DB_LIB_SOURCE_DIR} )
@@ -77,7 +77,7 @@ else()
 	add_executable( ${PROJECT_NAME} ${header_list} ${source_list} ${moc_list} ${generated_ui_list} ${generated_qrc_list} ${rc_list})
 endif()
 
-target_link_libraries( ${PROJECT_NAME} GLEW_LIB )
+target_link_libraries( ${PROJECT_NAME} "-lGLEW" )
 target_link_libraries( ${PROJECT_NAME} CC_FBO_LIB )
 target_link_libraries( ${PROJECT_NAME} CC_CORE_LIB )
 target_link_libraries( ${PROJECT_NAME} QCC_DB_LIB )
--- a/qCC/CMakeLists.txt
+++ b/qCC/CMakeLists.txt
@@ -78,7 +78,7 @@ else()
 	add_executable( ${PROJECT_NAME} ${header_list} ${source_list} ${moc_list} ${generated_ui_list} ${generated_qrc_list} ${rc_list} ${resource_list} ${txt_list} )
 endif()
 
-target_link_libraries( ${PROJECT_NAME} GLEW_LIB )
+target_link_libraries( ${PROJECT_NAME} "-lGLEW" )
 target_link_libraries( ${PROJECT_NAME} CC_FBO_LIB )
 target_link_libraries( ${PROJECT_NAME} CC_CORE_LIB )
 target_link_libraries( ${PROJECT_NAME} QCC_DB_LIB )
--- a/libs/CCFbo/CMakeLists.txt
+++ b/libs/CCFbo/CMakeLists.txt
@@ -11,7 +11,7 @@ endif()
 
 include_directories( ${EXTERNAL_LIBS_INCLUDE_DIR} )
 include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/include )
-include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
+# include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
 
 project( CC_FBO_LIB )
 
@@ -26,7 +26,7 @@ endif()
 
 add_library( ${PROJECT_NAME} STATIC ${header_list} ${source_list} )
 
-target_link_libraries( ${PROJECT_NAME} GLEW_LIB )
+target_link_libraries( ${PROJECT_NAME} "-lGLEW" )
 target_link_libraries( ${PROJECT_NAME} ${EXTERNAL_LIBS_LIBRARIES} )
 
 if ( USE_QT5 )
--- a/libs/qCC_db/CMakeLists.txt
+++ b/libs/qCC_db/CMakeLists.txt
@@ -11,7 +11,7 @@ endif()
 
 
 include_directories( ${EXTERNAL_LIBS_INCLUDE_DIR} )
-include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
+# include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
 include_directories( ${CC_CORE_LIB_SOURCE_DIR}/include )
 include_directories( ${CC_FBO_LIB_SOURCE_DIR}/include )
 if( MSVC )
@@ -32,7 +32,7 @@ endif()
 add_library( ${PROJECT_NAME} SHARED ${header_list} ${source_list} ${moc_list} )
 
 target_link_libraries( ${PROJECT_NAME} CC_CORE_LIB )
-target_link_libraries( ${PROJECT_NAME} GLEW_LIB )
+target_link_libraries( ${PROJECT_NAME} "-lGLEW" )
 target_link_libraries( ${PROJECT_NAME} CC_FBO_LIB )
 target_link_libraries( ${PROJECT_NAME} ${EXTERNAL_LIBS_LIBRARIES} )
 
--- a/libs/qCC_glWindow/CMakeLists.txt
+++ b/libs/qCC_glWindow/CMakeLists.txt
@@ -11,7 +11,7 @@ endif()
 
 include_directories( ${CMAKE_CURRENT_SOURCE_DIR} )
 include_directories( ${EXTERNAL_LIBS_INCLUDE_DIR} )
-include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
+# include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
 include_directories( ${CC_CORE_LIB_SOURCE_DIR}/include )
 include_directories( ${CC_FBO_LIB_SOURCE_DIR}/include )
 include_directories( ${QCC_DB_LIB_SOURCE_DIR} )
@@ -34,7 +34,7 @@ add_library( ${PROJECT_NAME} STATIC ${he
 
 target_link_libraries( ${PROJECT_NAME} CC_CORE_LIB )
 target_link_libraries( ${PROJECT_NAME} QCC_DB_LIB )
-target_link_libraries( ${PROJECT_NAME} GLEW_LIB )
+target_link_libraries( ${PROJECT_NAME} "-lGLEW" )
 target_link_libraries( ${PROJECT_NAME} CC_FBO_LIB )
 target_link_libraries( ${PROJECT_NAME} ${EXTERNAL_LIBS_LIBRARIES} )
 
--- a/libs/qCC_io/CMakeLists.txt
+++ b/libs/qCC_io/CMakeLists.txt
@@ -11,7 +11,7 @@ endif()
 
 include_directories( ${CMAKE_CURRENT_SOURCE_DIR} )
 include_directories( ${EXTERNAL_LIBS_INCLUDE_DIR} )
-include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
+# include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
 include_directories( ${CC_CORE_LIB_SOURCE_DIR}/include )
 include_directories( ${CC_FBO_LIB_SOURCE_DIR}/include )
 include_directories( ${QCC_DB_LIB_SOURCE_DIR} )
--- a/plugins/CMakePluginTpl.cmake
+++ b/plugins/CMakePluginTpl.cmake
@@ -3,7 +3,7 @@
 include_directories( ${CMAKE_CURRENT_SOURCE_DIR} )
 include_directories( ${CMAKE_CURRENT_BINARY_DIR} )
 include_directories( ${CloudComparePlugins_SOURCE_DIR} )
-include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
+# include_directories( ${GLEW_LIB_SOURCE_DIR}/include )
 include_directories( ${CC_FBO_LIB_SOURCE_DIR}/include )
 include_directories( ${CC_CORE_LIB_SOURCE_DIR}/include )
 include_directories( ${QCC_DB_LIB_SOURCE_DIR} )
@@ -60,7 +60,7 @@ else()
 	set_property( TARGET ${PROJECT_NAME} APPEND PROPERTY COMPILE_DEFINITIONS_RELEASE QT_NO_DEBUG)
 endif()
 
-target_link_libraries( ${PROJECT_NAME} GLEW_LIB )
+target_link_libraries( ${PROJECT_NAME} "-lGLEW" )
 target_link_libraries( ${PROJECT_NAME} CC_FBO_LIB )
 target_link_libraries( ${PROJECT_NAME} CC_CORE_LIB )
 target_link_libraries( ${PROJECT_NAME} QCC_DB_LIB )
