From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Fri, 15 Mar 2013 09:47:39 +0100
Subject: Fix local decimal separator
--- a/ccViewer/main.cpp
+++ b/ccViewer/main.cpp
@@ -47,6 +47,7 @@ public:
 	{
 		setOrganizationName("CCCorp");
 		setApplicationName("CloudCompareViewer");
+        std::setlocale(LC_NUMERIC, "C");
 #ifdef Q_OS_MAC
 		mViewer = NULL;
 
--- a/qCC/main.cpp
+++ b/qCC/main.cpp
@@ -53,6 +53,7 @@ public:
 	{
 		setOrganizationName("CCCorp");
 		setApplicationName("CloudCompare");
+        std::setlocale(LC_NUMERIC, "C");
 #ifdef Q_OS_MAC
 		// Mac OS X apps don't show icons in menus
 		setAttribute( Qt::AA_DontShowIconsInMenus );
@@ -152,7 +153,7 @@ int main(int argc, char **argv)
 
 			mainWindow->addToDB(filenames,UNKNOWN_FILE);
 		}
-		
+
 		if (splash)
 		{
             //we want the splash screen to be visible a minimum amount of time (1000 ms.)
