From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Fri, 15 Mar 2013 09:47:39 +0100
Subject: Fix local decimal separator
Index: cloudcompare-archeos/ccViewer/main.cpp
===================================================================
--- cloudcompare-archeos.orig/ccViewer/main.cpp	2014-08-03 18:01:14.257405245 +0200
+++ cloudcompare-archeos/ccViewer/main.cpp	2014-08-03 18:01:14.253405225 +0200
@@ -47,6 +47,7 @@
 	{
 		setOrganizationName("CCCorp");
 		setApplicationName("CloudCompareViewer");
+        std::setlocale(LC_NUMERIC, "C");
 #ifdef Q_OS_MAC
 		mViewer = NULL;
 
Index: cloudcompare-archeos/qCC/main.cpp
===================================================================
--- cloudcompare-archeos.orig/qCC/main.cpp	2014-08-03 18:01:14.257405245 +0200
+++ cloudcompare-archeos/qCC/main.cpp	2014-08-03 18:01:14.253405225 +0200
@@ -53,6 +53,7 @@
 	{
 		setOrganizationName("CCCorp");
 		setApplicationName("CloudCompare");
+        std::setlocale(LC_NUMERIC, "C");
 #ifdef Q_OS_MAC
 		// Mac OS X apps don't show icons in menus
 		setAttribute( Qt::AA_DontShowIconsInMenus );
@@ -152,7 +153,7 @@
 
 			mainWindow->addToDB(filenames,UNKNOWN_FILE);
 		}
-		
+
 		if (splash)
 		{
 			//we want the splash screen to be visible a minimum amount of time (1000 ms.)
