--- a/qCC/plugins/qHPR/qHPR.cpp
+++ b/qCC/plugins/qHPR/qHPR.cpp
@@ -28,7 +28,7 @@
 #include <ccProgressDialog.h>
 
 //qCC
-#include <ccGLWindow.h>
+#include "../../ccGLWindow.h"
 
 //CCLib
 #include <CloudSamplingTools.h>
--- a/qCC/CMakeLists.txt
+++ b/qCC/CMakeLists.txt
@@ -1,5 +1,12 @@
 cmake_minimum_required(VERSION 2.8)
 
+project( CloudCompare )
+
+set( VERSION_MAJOR 2 )
+set( VERSION_MINOR 4 )
+set( VERSION_PATCH 0 )
+
+
 if ( UNIX )
 	set( CloudCompare_dest_release /usr/bin ) #default destination: /usr/bin
 	set( CloudCompare_dest_debug bin/debug )
@@ -48,12 +55,6 @@ include_directories( ${CMAKE_CURRENT_BIN
 include_directories( ${LIBLAS_INCLUDE_DIR} )
 include_directories( ${XIOT_INCLUDE_DIR} )
 
-project( CloudCompare )
-
-set( VERSION_MAJOR 2 )
-set( VERSION_MINOR 4 )
-set( VERSION_PATCH 0 )
-
 # TODO: use qxt as a standalone library!
 set( QXT_HEADERS ../libs/qxt/qxtspanslider.h ../libs/qxt/qxtspanslider_p.h )
 set( QXT_SOURCES ../libs/qxt/qxtspanslider.cpp )
