I/
+++ b/qCC/plugins/qBlur/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8)
 
-option( INSTALL_QBLUR_PLUGIN "Check to install qBlur plugin" OFF )
+option( INSTALL_QBLUR_PLUGIN "Check to install qBlur plugin" ON )
 
 if (INSTALL_QBLUR_PLUGIN)
 
--- a/qCC/plugins/qHPR/CMakeLists.txt
+++ b/qCC/plugins/qHPR/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8)
 
-option( INSTALL_QHPR_PLUGIN "Check to install qHPR plugin" OFF )
+option( INSTALL_QHPR_PLUGIN "Check to install qHPR plugin" ON )
 
 if (INSTALL_QHPR_PLUGIN)
 
@@ -13,5 +13,6 @@ include( ../CMakePluginTpl.cmake )
 
 target_link_libraries( ${PROJECT_NAME} qhull )
 include_directories( ${QHULL_LIB_INCLUDE_DIR} )
+include_directories( ${qCC_SOURCE_DIR} )
 
 endif()
--- a/qCC/plugins/qPoissonRecon/CMakeLists.txt
+++ b/qCC/plugins/qPoissonRecon/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8)
 
-option( INSTALL_QPOISSON_RECON_PLUGIN "Check to install qPoissonRecon plugin" OFF )
+option( INSTALL_QPOISSON_RECON_PLUGIN "Check to install qPoissonRecon plugin" ON )
 
 if (INSTALL_QPOISSON_RECON_PLUGIN)
 
--- a/qCC/plugins/qEDL/CMakeLists.txt
+++ b/qCC/plugins/qEDL/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8)
 
-option( INSTALL_QEDL_PLUGIN "Check to install qEDL plugin" OFF )
+option( INSTALL_QEDL_PLUGIN "Check to install qEDL plugin" ON )
 
 if (INSTALL_QEDL_PLUGIN)
 
--- a/qCC/plugins/qKinect/CMakeLists.txt
+++ b/qCC/plugins/qKinect/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8)
 
-option( INSTALL_QKINECT_PLUGIN "Check to install qKINECT plugin" OFF )
+option( INSTALL_QKINECT_PLUGIN "Check to install qKINECT plugin" ON )
 
 if (INSTALL_QKINECT_PLUGIN)
 
@@ -8,8 +8,8 @@ if (INSTALL_QKINECT_PLUGIN)
 project( QKINECT_PLUGIN_DLL )
 
 # libfreenect is needed to build library
-set( LIBFREENECT_INCLUDE_DIR "" CACHE PATH "Include directory of libfreenect" )
-set( LIBFREENECT_LIBRARY_FILE "" CACHE FILE "Library file of libfreenect" )
+set( LIBFREENECT_INCLUDE_DIR "/usr/include" CACHE PATH "Include directory of libfreenect" )
+set( LIBFREENECT_LIBRARY_FILE "-lfreenect" CACHE FILE "Library file of libfreenect" )
 if (WIN32)
 	set( LIBFREENECT_SHARED_LIBRARY_FILE "" CACHE FILE "Shared library file of libfreenect (dll)" )
 endif()
--- a/qCC/plugins/qSSAO/CMakeLists.txt
+++ b/qCC/plugins/qSSAO/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8)
 
-option( INSTALL_QSSAO_PLUGIN "Check to install qSSAO plugin" OFF )
+option( INSTALL_QSSAO_PLUGIN "Check to install qSSAO plugin" ON )
 
 if (INSTALL_QSSAO_PLUGIN)
 
--- a/qCC/plugins/qHPR/qHPR.cpp
+++ b/qCC/plugins/qHPR/qHPR.cpp
@@ -28,7 +28,7 @@
 #include <ccProgressDialog.h>
 
 //qCC
-#include <ccGLWindow.h>
+#include "../../ccGLWindow.h"
 
 //CCLib
 #include <CloudSamplingTools.h>
--- a/qCC/plugins/qPCV/CMakeLists.txt
+++ b/qCC/plugins/qPCV/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8)
 
-option( INSTALL_QPCV_PLUGIN "Check to install qPCV plugin" OFF )
+option( INSTALL_QPCV_PLUGIN "Check to install qPCV plugin" ON )
 
 if (INSTALL_QPCV_PLUGIN)
 
--- a/qCC/plugins/qPCV/qPCV.cpp
+++ b/qCC/plugins/qPCV/qPCV.cpp
@@ -32,8 +32,8 @@
 #include <ccProgressDialog.h>
 
 //qCC
-#include <ccCommon.h>
-#include <ccDBRoot.h>
+#include "../../ccCommon.h"
+#include "../../db_tree/ccDBRoot.h"
 
 qPCV::qPCV(QObject* parent/*=0*/)
 	: QObject(parent)
