--- a/libs/qCC_io/DxfFilter.cpp
+++ b/libs/qCC_io/DxfFilter.cpp
@@ -586,8 +586,8 @@ CC_FILE_ERROR DxfFilter::saveToFile(ccHO
 	dw->tableEnd();
 
 	//Writing Various Other Tables
-	//dxf.writeStyle(*dw); //DXFLIB V2.5
-	dxf.writeStyle(*dw,DL_StyleData("Standard",0,0.0,0.75,0.0,0,2.5,"txt","")); //DXFLIB V3.3
+	dxf.writeStyle(*dw); //DXFLIB V2.5
+	// dxf.writeStyle(*dw,DL_StyleData("Standard",0,0.0,0.75,0.0,0,2.5,"txt","")); //DXFLIB V3.3
 	dxf.writeView(*dw);
 	dxf.writeUcs(*dw);
 
--- a/plugins/qSRA/dxfProfilesExporter.cpp
+++ b/plugins/qSRA/dxfProfilesExporter.cpp
@@ -223,8 +223,8 @@ bool DxfProfilesExporter::SaveVerticalPr
 	dw->tableEnd();
 
 	//Writing Various Other Tables
-	//dxf.writeStyle(*dw); //DXFLIB V2.5
-	dxf.writeStyle(*dw,DL_StyleData("Standard",0,0.0,0.75,0.0,0,2.5,"txt","")); //DXFLIB V3.3
+	dxf.writeStyle(*dw); //DXFLIB V2.5
+	//dxf.writeStyle(*dw,DL_StyleData("Standard",0,0.0,0.75,0.0,0,2.5,"txt","")); //DXFLIB V3.3
 	dxf.writeView(*dw);
 	dxf.writeUcs(*dw);
 
@@ -705,8 +705,8 @@ bool DxfProfilesExporter::SaveHorizontal
 	dw->tableEnd();
 
 	//Writing Various Other Tables
-	//dxf.writeStyle(*dw); //DXFLIB V2.5
-	dxf.writeStyle(*dw,DL_StyleData("Standard",0,0.0,0.75,0.0,0,2.5,"txt","")); //DXFLIB V3.3
+	dxf.writeStyle(*dw); //DXFLIB V2.5
+	//dxf.writeStyle(*dw,DL_StyleData("Standard",0,0.0,0.75,0.0,0,2.5,"txt","")); //DXFLIB V3.3
 	dxf.writeView(*dw);
 	dxf.writeUcs(*dw);
 
--- a/contrib/DxfLibSupport.cmake
+++ b/contrib/DxfLibSupport.cmake
@@ -9,7 +9,7 @@ if( ${OPTION_USE_DXF_LIB} )
 	if ( NOT DXF_LIB_SRC_DIR )
 		message( SEND_ERROR "No DXF lib source dir specified (DXF_LIB_SRC_DIR)" )
 	else()
-		add_subdirectory(contrib/dxflib)
+        # add_subdirectory(contrib/dxflib)
 		include_directories( ${DXF_LIB_SRC_DIR} )
 	endif()
 
@@ -20,7 +20,7 @@ function( target_link_DXFLIB ) # 2 argum
 
 if( ${OPTION_USE_DXF_LIB} )
 
-	target_link_libraries( ${PROJECT_NAME} DXF_LIB )	
+	target_link_libraries( ${PROJECT_NAME} "-ldxflib" )	
 	set_property( TARGET ${PROJECT_NAME} APPEND PROPERTY COMPILE_DEFINITIONS CC_DXF_SUPPORT )
 
 endif()
